sklearn/linear_model/tests/test_ransac.py

```diff
@@ -353,3 +353,14 @@ def test_ransac_dynamic_max_trials():
     ransac_estimator = RANSACRegressor(base_estimator, min_samples=2,
                                        stop_probability=1.1)
     assert_raises(ValueError, ransac_estimator.fit, X, y)
+
+def test_ransac_fit_sample_weight():
+    ransac_estimator = RANSACRegressor()
+    n_samples = y.shape[0]
+    weights = np.ones(n_samples)
+    ransac_estimator.fit(X, y, weights)
+    assert_equal(ransac_estimator.inlier_mask_.shape[0], n_samples) #sanity check
```

Comment:
Use `assert_array_equal` to check that the masks are the same. Also check ransac_estimator.estimator.coef_ as well